### ISSUE TITLE
Show "Error in digesting: ..: Permission denied"

### ISSUE THREAD

**Issue by ymhuang0808** [EugenMayer/docker-sync#148]
Thanks docker-sync for improving the Docker performance. 

I encounter permission problem when the database data is changed. It shows the following message:
```
Synchronization complete at 09:44:54  (0 item transferred, 1 skipped, 0 failed)
  skipped: ibdata1 (Error in digesting /myproject/mariadb/data/ibdata1:
/myproject/mariadb/data/ibdata1: Permission denied)
```
Here is my configuration and docker-sync version.
```yml
options:
  compose-file-path: './myproject/docker/docker-compose.yml'
  compose-dev-file-path: './myproject/docker/docker-compose-dev.yml'
  verbose: true
syncs:
  myproject-mariadb-data-unison-sync:
    src: './myproject/docker/data'
    dest: '/myproject/mariadb/data'
    sync_strategy: 'unison'
```

```
docker-sync (0.1.2)
```

May I configure the `sync_user` or `sync_userid` in `docker-sync.yml`?

**Comment by EugenMayer** [comment]
@ymhuang0808 are you writing on a MYSQL data dir using 2 different mysql-instances at the same time? In general, syncing mysql data dir should be a bad idea and could not work out of different reasons - such databases are very picky when it comes down to there filesystem / IO to optimize performance.
You might check if google tells you something about unison + mysql data dir

@mickaelperrin any hints here?

**Comment by mickaelperrin** [comment]
Indeed, I wouldn't recommend also to sync database files, especially with unison. If you want to persist your data across container wipe, simply use a dedicated volume for that, and use sql dumps for backups.

If you still want to try unison for this purpose, you indeed need to set the appropriate `sync_user=mysql`and `sync_userid=999` for official mariadb image.

**Comment by ymhuang0808** [comment]
@EugenMayer @mickaelperrin Sorry for the late reply.

There is only one mysql-instance writing a data directory at the same time. Does it affect the database I/O, if the database files are synced over unison?

Yes, I would like to persist the database data after a container is deleted.

**Comment by danielvdspuy** [comment]
@mickaelperrin – When you say use a dedicated volume, do you mean a regular Docker volume without docker-sync? Or do you mean use rsync instead of unison?

**Comment by mickaelperrin** [comment]
@danielvdspuy I mean a regular docker volume

**Comment by henscu** [comment]
MYSQL is a very common use case for Docker (eg in WordPress). I only realised that sync for MYSQL would be a problem when I saw the latest  issue #263 and did a search for MYSQl in the issues which led me to this one. There is nothing in the Wiki about this specific use case.

I am working on a page for the Wiki about setting up WordPress using docker-sync. I can add this info to help others avoid my mistake.
